internal/crypto/Cargo.toml

```diff
@@ -31,15 +31,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 json_schema = ["dep:schemars"]
-rust_native_crypto = [
-    "dep:const-oid",
-    "dep:der",
-    "dep:ed25519-dalek",
-    "dep:num-bigint-dig",
-    "dep:pkcs1",
-    "dep:rsa",
-    "dep:spki",
-]
+openssl = ["dep:openssl"]
```

Comment:
it looks like this flips things around so that we always build with rust native crypto and so openssl builds would get larger than they are now. Do we know how much larger?
I'd still like to have a no-crypto option for embedded builds that don't need to validate, but we can work on that later.

Reply:
Since I added additional crates to handle PEM parsing for es512 it's only about 100KB